scripts/build/TestPlatform.Settings.targets

```diff
@@ -52,6 +52,11 @@
         <GenerateDocumentationFile>false</GenerateDocumentationFile>
       </PropertyGroup>
 
+      <!-- Test projects are not discovered in test window without test container capability -->
```

Comment:
Please remove `<Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />`. I think tests should get discovered.